projects/sandbox/train/train/train.py

```diff
@@ -47,10 +48,9 @@ def main(
     kernel_length: float,
     sample_rate: float,
     batch_size: int,
-    mean_snr: float = 8,
-    std_snr: float = 4,
-    min_snr: Optional[float] = None,
-    highpass: Optional[float] = None,
+    highpass: float,
+    train_val_start: float,
```

Comment:
I think the issue with making the validation start and stop times the parameters here is that it would need to change these with each choice of `valid_frac` (and would really obviate that parameter altogether). I think if you just specify the start of the training period, you can infer the duration from the length of the background and the sample rate, and then from the `valid_frac` you can infer where the valid period begins.

In fact, now that I think about it, we save `t0` with the training data, so we can probably infer all of this without any additional parameters, right?

Reply:
`train_val_start` and `train_val_stop` refers to the start time of the combined training and validation period. The logic for calculating the validation start is still there. I renamed things for clarity since `train_start` and `train_stop` actually referred to training _and_ validation start and stop

Reply:
Okay took care of this, give it another look. 

Background is now saved in one file with the `ifo` as the dataset key, and `t0` as an attribute. `ifos`, `train_val_start` and `train_val_stop` are inferred from this file.